DisCatSharp/Entities/User/DiscordUser.cs

```diff
@@ -154,6 +161,13 @@ public string BannerUrl
 	public string AvatarUrl
 		=> string.IsNullOrWhiteSpace(this.AvatarHash) ? this.DefaultAvatarUrl : $"{DiscordDomain.GetDomain(CoreDomain.DiscordCdn).Url}{Endpoints.AVATARS}/{this.Id.ToString(CultureInfo.InvariantCulture)}/{this.AvatarHash}.{(this.AvatarHash.StartsWith("a_") ? "gif" : "png")}?size=1024";
 
+	/// <summary>
+	/// Gets the user's avatar decoration URL.s
```

Comment:
```suggestion
	/// Gets the user's avatar decoration url.
```
Fix that in the upper function too

Reply:
Done